src/components/counter-box/CounterBox.tsx

```diff
@@ -0,0 +1,71 @@
+import {StyleSheet, Theme, useTheme} from '@atb/theme';
+import {View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import React from 'react';
+import {iconSizes} from '@atb-as/theme';
+import {TextNames} from '@atb/theme/colors';
+
+interface StylingCounter {
+  padding: number;
+  lineHeight: number;
+  type: TextNames;
+}
+export const CounterBox = ({
+  count,
+  size = 'normal',
+  accessibilityLabel,
+}: {
+  count: number;
+  size?: keyof Theme['icon']['size'];
+  accessibilityLabel?: string;
```

Comment:
I think it is better to include `AccessibilityProps` here instead of just `accessibilityLabel`. So the typing would be:
```ts
{
  count: number;
  size?: keyof Theme['icon']['size'];
} & AccessibilityProps
```

Look at `TouchableOpacityOrView` for an example.